src/streaming/controllers/CmcdController.js

```diff
@@ -139,16 +139,16 @@ function CmcdController() {
     }
     
     function _onStateChange(state) {
+        internalData.sta = state;
         const targets = settings.get().streaming.cmcd.targets;
         const eventModeTargets = targets.filter((target) => target.cmcdMode === Constants.CMCD_MODE.EVENT);
         
         if (eventModeTargets.length === 0) {
             return;
         }
-        
+
         const cmcdData = _getGenericCmcdData();
         cmcdData.e = state;
```

Comment:
Create convertor from player state to cmcd reporting envent

```suggestion
        cmcdData.e = convertPlayerStateToCmcdEvent(state);
```